rpcs3/Emu/System.h

```diff
@@ -626,6 +626,7 @@ struct cfg_root : cfg::node
 		cfg::_bool use_native_interface{ this, "Use native user interface", true };
 		cfg::string gdb_server{this, "GDB Server", "127.0.0.1:2345"};
 		cfg::_bool silence_all_logs{this, "Silence All Logs", false, false};
+		cfg::string title_format{this, "Window Title Format", "%R | %V | %T [%t]", true};
```

Comment:
This setting is not dynamic, because you use it to init the window title in the constructor.